### ISSUE TITLE
from binary (elf file) to LLVM bitcode

### ISSUE THREAD

**Issue by nisrine** [libcpu/libcpu#4]
Hi,
I would like to know if I can use this tool to generate the llvm bitcode from a binary file?if yes could you please tell me how.
thanks.

**Comment by ismaell** [comment]
Yes, it does code lifting to do it's job, but you'll have to figure it out...
